src/services/esri-feature/types.d.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  esriGeometryPoint,
+  esriGeometryMultipoint,
+  esriGeometryPolyline,
+  esriGeometryPolygon,
+  esriGeometryEnvelope,
+} from '@esri/arcgis-rest-feature-layer';
+import type { IFeature } from '@esri/arcgis-rest-request';
+
+interface Feat extends IFeature {
```

Comment:
Maybe it will be better to create a subfolder on the definitions folder that already exists (maybe you have to rebase with develop). This way there will be much easier to access